libnopegl/src/bindgroup.h

```diff
@@ -81,16 +83,17 @@ struct buffer_binding {
 };
 
 struct bindgroup_params {
-    const struct bindgroup_layout *layout;
+    struct bindgroup_layout *layout;
```

Comment:
is this related to this commit? (same below)

Reply:
Old chunk from a previous iteration. Removed.